js/values/AssessmentResult.js

```diff
@@ -0,0 +1,121 @@
+var inRange = require( "lodash/number/inRange" );
```

Comment:
This does too much for a value object.

If this is still an iteration, nvm me. But we should be going towards something final very soon.